enterprise/backend/src/metabase_enterprise/serialization/v2/extract.clj

```diff
@@ -66,9 +66,10 @@
   (loop [to-chase (set targets)
          chased   #{}]
     (let [[m i :as item] (first to-chase)
-          desc           (serdes/descendants m i)
-          chased         (conj chased item)
-          to-chase       (set/union (disj to-chase item) (set/difference desc chased))]
+          asc                 (serdes/ascendants m i)
+          desc                (serdes/descendants m i)
+          chased              (-> (conj chased item) (into asc))
+          to-chase            (set/union (disj to-chase item) (set/difference desc chased))]
```

Comment:
This feels a little arbitrary that ascendants aren't chased but descendants are. I was imagining that ascendants would be chased for ascendants but at least their descendants wouldn't be. Anyway, this works fine because for collections getting the full path of parents is very quick and easy, so chasing is unnecessary.

The difference between ascendants and descendants should be explained in the docstring of this function at least.